node/test/Test/Pos/Block/Identity/BinarySpec.hs

```diff
@@ -66,8 +65,9 @@ spec = giveStaticConsts $ describe "Block types" $ do
                 describe "Body" $ do
                     binaryTest @(BT.Body (BT.MainBlockchain SscNistBeacon))
                     binaryTest @(BT.Body (BT.MainBlockchain SscGodTossing))
-                binaryTest @BT.MainExtraHeaderData
-                binaryTest @BT.MainExtraBodyData
                 describe "MainToSign" $ do
                     binaryTest @(BT.MainToSign SscNistBeacon)
```

Comment:
I have a feeling that we can delete `SscNistBeacon` tests, but it might be subjective and not for this PR :)